apps/pie-microsite/src/_11ty/shortcodes/notification.js

```diff
@@ -65,6 +66,6 @@ module.exports = function (config) {
 
     return `<aside class="${contextClass} c-notification" style="--bg-colour: ${bgColour}">
       ${svg}
-      <p class="c-notification-message">${config.message}</p>
+      <p class="c-notification-message">${markdownFilter(config.message, true)}</p>
```

Comment:
can we update the jsdoc annotation for `config.message` to say it supports markdown? There's an example in the `contentPageImage.js` shortcode